inspec.yml

```diff
@@ -109,20 +110,20 @@ inputs:
     required: true
     sensitive: true
 
-  # not used in controls
-  - name: pg_syslog_owner
-    description: "The syslog process owner that the progress logs should use"
-    type: string
-    value: "postgres" 
-    required: true
-    sensitive: true
-
-    # not used in controls
-  - name: pg_syslog_facility
-    description: "The syslog facility that postgres should be set to use for logging"
-    type: array
-    value:
-      - local0
+  # for possible future use

```

Comment:
I would just delete these commented out values in general to avoid confusion. They can be found again in the git history if needed.